### PR TITLE
fix export in filter data with form

### PR DESCRIPTION
**Pull request by drillinP**
* Ticket : #2308

### CODE REVIEW

File: lizmap/www/assets/js/filter.js [hunk 1]
```diff
@@ -47,7 +47,9 @@ var lizLayerFilterTool = function() {
             html+= '<br/><button id="liz-filter-zoom" class="btn btn-mini btn-primary" title="'+lizDict['filter.btn.zoom.title']+'">'+lizDict['filter.btn.zoom.label']+'</button>';
 
             // Add export button
-            html+= '&nbsp;&nbsp;<button id="liz-filter-export" class="btn btn-mini btn-primary" title="'+lizDict['filter.btn.export.title']+'">'+lizDict['filter.btn.export.label']+'</button>';
+            if (lizMap.getVectorLayerResultFormat().includes('ODS')) {
+                html+= '&nbsp;&nbsp;<button id="liz-filter-export" class="btn btn-mini btn-primary" title="'+lizDict['filter.btn.export.title']+'">'+lizDict['filter.btn.export.label']+'</button>';
+	    }
```
Comment (rldhont) [inline lizmap/www/assets/js/filter.js:52]: indent

**Review by rldhont — CHANGES_REQUESTED** [review]

File: lizmap/www/assets/js/filter.js (hunk 1 above)
Comment (drillinP) [reply lizmap/www/assets/js/filter.js]: @rldhont it's ok ?

**Review by nboisteault — APPROVED** [review]
LGTM

File: lizmap/www/assets/js/filter.js [hunk 2]
```diff
@@ -47,7 +47,9 @@ var lizLayerFilterTool = function() {
             html+= '<br/><button id="liz-filter-zoom" class="btn btn-mini btn-primary" title="'+lizDict['filter.btn.zoom.title']+'">'+lizDict['filter.btn.zoom.label']+'</button>';
 
             // Add export button
-            html+= '&nbsp;&nbsp;<button id="liz-filter-export" class="btn btn-mini btn-primary" title="'+lizDict['filter.btn.export.title']+'">'+lizDict['filter.btn.export.label']+'</button>';
+            if (lizMap.getVectorLayerResultFormat().includes('ODS')) {
+                html+= '&nbsp;&nbsp;<button id="liz-filter-export" class="btn btn-mini btn-primary" title="'+lizDict['filter.btn.export.title']+'">'+lizDict['filter.btn.export.label']+'</button>';
+	        }
```
Comment (rldhont) [inline lizmap/www/assets/js/filter.js:52]: still indent error

**Review by rldhont — CHANGES_REQUESTED** [review]

**Comment by rldhont** [comment]
Changelog

**Review by rldhont — APPROVED** [review]

**Pull request merged**